api/Parser.cpp

```diff
@@ -4,6 +4,29 @@
 namespace
 {
 
+const nlohmann::json EMPTY_JSON_ARRAY = nlohmann::json::array();
```

Comment:
I do not think this alias is necessary. Or do we run into compiler problems at line 12? At line 12 it is still obvious it is an empty array.

Reply:
I changed the place to be more clear the purpose of that variable. Now is a `static` variable scoped in `optionalJsonArray` function